tests/integration/test_list_simulations.py

```diff
@@ -0,0 +1,93 @@
+import copy
+import json
+import logging
+import platform
+import pytest
+from time import sleep
+from flask import Flask
+
+from yaptide.persistence.database import db
+
+
+# @pytest.mark.skip(reason="no way of currently testing this")
+def test_list_simulations(celery_app, 
+                                   celery_worker, 
+                                   client_fixture: Flask, 
+                                   db_good_username: str, 
+                                   db_good_password: str, 
+                                   payload_editor_dict_data: dict,
+                                   add_directory_to_path,
+                                   shieldhit_demo_binary):
+    """Test we can run simulations"""
+    client_fixture.put("/auth/register",
+                       data=json.dumps(dict(username=db_good_username, password=db_good_password)),
+                       content_type='application/json')
+    resp = client_fixture.post("/auth/login",
+                               data=json.dumps(dict(username=db_good_username, password=db_good_password)),
+                               content_type='application/json')
+
+    assert resp.status_code == 202  # skipcq: BAN-B101
+    assert resp.headers['Set-Cookie']  # skipcq: BAN-B101
+
+    payload_dict = copy.deepcopy(payload_editor_dict_data)
+
+    # limit the particle numbers to get faster results
+    payload_dict["ntasks"] = 1
+    payload_dict["input_json"]["beam"]["numberOfParticles"] = 12
+
+    if platform.system() == "Windows":
+        payload_dict["input_json"]["detectManager"]["filters"] = []
+        payload_dict["input_json"]["detectManager"]["detectGeometries"] = [payload_dict["input_json"]["detectManager"]["detectGeometries"][0]]
+        payload_dict["input_json"]["scoringManager"]["scoringOutputs"] = [payload_dict["input_json"]["scoringManager"]["scoringOutputs"][0]]
+        for output in payload_dict["input_json"]["scoringManager"]["scoringOutputs"]:
+            for quantity in output["quantities"]["active"]:
+                if "filter" in quantity:
+                    del quantity["filter"]
+
+    logging.info("Sending job submition request on /jobs/direct endpoint")
+    number_of_simulations = 8
```

Comment:
Add a comment explaining why do you submit so many simulations here. What is being tested ?